styles/style.css

```diff
@@ -374,7 +374,7 @@ body {
 
```

Comment:
- [ ] Kindly fix your images to appear on the web

![image](https://user-images.githubusercontent.com/93630700/211115252-5cd377ce-aa82-4b7f-8879-55f84d74cd9b.png)